docs/2_2_common_mechanisms.adoc

```diff
@@ -1161,30 +1161,7 @@ Both functions have the same arguments:
 * `nSensitivity` contains the length of `sensitivity`.
 
 _[Note that array variables will be serialized, so `nSeed` is only equal to `nKnowns` in the case of directional derivatives (resp., equal to `nUnknowns` in the case of adjoint derivatives) if all value references of `knowns` (resp., `unknowns`) point to scalar variables._
-_Likewise `nSensitivity` is only equal to `nUnknowns` (resp., `nKnowns`) if all value references of `unknowns` (resp., `knowns`) point to scalar variables._
-
-_Note that it is the partial derivative and not the total derivative that is considered here._
-_When computing the partial derivative with respect to the independent variable_ latexmath:[t] _of the function_
-
-[latexmath]
-++++
-\mathbf{f}(\mathbf{x}, \mathbf{\dot{x}}, t)
-++++
-
-_its partial derivative with respect to_ latexmath:[t] _will be_
-
-[latexmath]
-++++
-\frac{\partial \mathbf{f}}{\partial t}
-++++
-
-_and not_
-
-[latexmath]
-++++
-\frac{\partial \mathbf{f}}{\partial t} + \frac{\partial \mathbf{f}}{\partial \mathbf{x}} \mathbf{\dot{x}}+\frac{\partial \mathbf{f}}{\partial \mathbf{\dot{x}}} \mathbf{\ddot{x}}
-++++
-_]_
```

Comment:
Why do we remove the explanation that the partial and not the total derivative w.r.t time is meant? Is it obvious or explained somewhere else?

Reply:
This was requested by @klausschuch and @masoud-najafi  as they thought it was more confusing that we added this section than leaving it out

Reply:
But if there are different opinions on this I can return the section again, I added it before because I thought it helped but accepted the removal since others thought it was more confusing